platform/viewer/cypress/integration/common/OHIFCornerstoneToolbar.spec.js

```diff
@@ -216,26 +219,49 @@ describe('OHIF Cornerstone Toolbar', () => {
       .click();
 
     let expectedText = 'Img: 1 1/26';
-    cy.get('@viewportInfoBottomLeft').should('not.have.text', expectedText);
+    cy.get('@viewportInfoBottomLeft', { timeout: 15000 }).should(
```

Comment:
Is there a reason these each need 15 second timeouts? That seems really high.

Reply:
Yeah, unfortunately, every interaction with cine controls demands a new image to load in the viewport. Most of the times it is taking a long time to fully load the image.